src/methods/tx/_events.ts

```diff
@@ -241,6 +245,23 @@ export const processChainEvent = (
          *                     [who, assets, sell or buy, intention id, error detail]
          */
         if (Array.isArray(parsedData)) {
+          const dispatchError = parsedData[4];
+          let errorDetails = dispatchError.toString();
+
+          if (parsedData[4].module !== undefined) {
+            const { documentation, section, name } = api.registry.findMetaError(
+              {
+                error: new BN(dispatchError.module.error),
```

Comment:
Do we have guarantee that module.error is always number? What happens if it is string value?

Reply:
I've added value check so now value must be a number.